src/Entity/Storage/AppStorage.php

```diff
@@ -94,6 +94,27 @@ public function loadUnchanged($id) {
     return parent::loadUnchanged($id);
   }
 
+  /**
+   * Load unchanged for when $id is an App id.
+   *
+   * This function is more efficient than loadUnchanged,
+   * because it does not need to cover the case
+   * when loading is done by App name.
+   *
+   * @param string $id
+   *   App id.
+   *
+   * @return \Drupal\Core\Entity\EntityInterface|mixed|null
+   *   The unchanged entity, or NULL if the entity cannot be loaded.
+   */
+  public function loadUnchangedById($id) {
```

Comment:
Missing parameter typehint (string) and return typehint: this is either null or AppInterface.

Reply:
added typehints; now the only function that does it in AppStorage.